extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -387,6 +428,52 @@ export class Visibility {
     s[MAX_VISIBLE] = s[MAX_VISIBLE] ? Math.max(s[MAX_VISIBLE], visible) : -1;
     s[LAST_VISIBLE_TIME] = Date.now() - s[TIME_LOADED];
   }
+
+  /**
+   * Sets variable values for callback. Cleans up existing values.
+   * @param {Object<string, *>} state The state object to populate
+   * @param {!LayoutRect} rb Bounds of Root object. (the viewport in this case)
+   * @param {!LayoutRect} br The bounding rectangle for the element
+   * @param {!LayoutRect} ir The intersection between element and the viewport
+   * @private
+   */
+  prepareStateForCallback_(state, rb, br, ir) {
+    const perf = this.win_.performance;
+    state[ELEMENT_X] = rb.left + br.left;
+    state[ELEMENT_Y] = rb.top + br.top;
+    state[ELEMENT_WIDTH] = br.width;
+    state[ELEMENT_HEIGHT] = br.height;
+    state[TOTAL_TIME] = perf && perf.timing && perf.timing.domInteractive
+        ? Date.now() - perf.timing.domInteractive
+        : '';
+
+    // Calculate the amount element visible at the time page was loaded. To do
+    // this, assume that the page is scrolled all the way to top.
+    const viewportRect = {top: 0, height: rb.height, left: 0, width: rb.width};
+    const elementRect = {top: ir.top, left: ir.left, width: br.width,
+      height: br.height};
+    const intersection = rectIntersection(viewportRect, elementRect);
+    state[LOAD_TIME_VISIBILITY] = intersection != null
+        ? Math.round(intersection.width * intersection.height * 10000
```

Comment:
This allows for two significant digits after decimal. So `12.344` becomes `12.34` and `12.346` becomes `12.35`